src/utils/SetupUtil.js

```diff
@@ -56,64 +56,56 @@ export default {
     let machineVersion = null;
     while (true) {
       try {
-        setupServerActions.started({started: false});
+        let ip = null;
```

Comment:
Well I would say that Linux is the only platform where you don't need setup. It make more sense to create a function for OSX and Windows and call it when not running a Linux box.

As you can see in the code: 

```javascript
        ...
        if (util.isLinux()) {
          ip = 'localhost';
        } else {
          ...
        }
```

we just set the `host` attribute, that's the Linux setup.